### ISSUE TITLE
Code Review - Abhay Sarda

### ISSUE THREAD

**Issue by abhaysarda** [leonjia0112/ec601_passenger_screening#14]
Hi Guys,

Because Shengye and Chen have already mentioned on the well written code and pep8 score, etc, I won't spend any time on those issues. 

I noticed that you mentioned the accuracy in the main readme file. Is the accuracy mentioned in the training part of readme for the AlexNet or VGGNet model? Since you mentioned that it was painfully slow , did you try using GoogLeNet or any other techniques? What is the average time for each image on the SCC vs your computer?

One thing which is confusing about seeing the project is how much of the data was available from the 
start, by Kaggle, and how much is the new work. But I guess that is one of Github's shortcomings,
that it is less visible to people new to the work. Also, I noticed that the credits were generally given for the files in the python scripts where needed. But for the body detection part, I wasn't sure if you chose those regions, or Kaggle gave the specifications.

Another minor edit is that the SCC.log file has failed test results. Since the keras load
failed, and the file ends there. I am guessing you ran tests but didn't need to update SCC.log. So maybe the file can be deleted?

The readme file in brances backend_tsy, Front end Web Dev, PeiJia branch and the Flask Backend are mostly invalid, since all of them have the same content about image files and data type. Of course documentation is a end-of-development thing, but maybe you can complete it when you can. 

Also as a final question, are you displaying the final results on a web page? Is that the reason you needed Flask?

While going through the Kaggle discussion, I noticed that you guys aren't actually eligible for the prize money, being foreign students and also that the competition ends in 6 days, so I hope you finish in time, and best of luck!

I am sure you guys learnt a lot, so well done!

Warm regards,
Abhay

**Comment by adityachamp** [comment]
Hi Abhay,

Thanks for your genuine feedback. So here's what we've got, firstly, yes, we did consider our options while selecting our Neural Network and we deliberately chose VGGNet over GoogLeNet. Primarily because its generally considered more accurate and it extracts features that are more general and effective for datasets (You can look it up). Secondly, the dataset is so huge that it simply doesn't run on our ubuntu or windows machines. After a few epochs, the system simply crashes. Because of which we used BU Cluster.

Answering your next question, yes, Kaggle provided is the dataset with zone markings and segregations. Just to give you the exact figures, they provided us roughly 18k images, 1/10 th of which comprised of a potential threat and each body divided into 17 zones.

We have deleted the SCC.log files with failed results. Thanks for informing. Finally, yes we are displaying the results on our webpage and hence we have used Flask framework.

Thanks for your feedback. Good Luck.

Best,
Passenger Screening Team
